core/classes/Core/Paginator.php

```diff
@@ -179,17 +179,4 @@ public function generate(int $links, string $href = '?'): string {
         return $html;
     }
 
-    /**
-     * Set values of instance variables, alternative function (as they are set in getLimited()).
-     * @deprecated Not used internally. Will be removed in 2.1.0
-     *
-     * @param int $total
-     * @param int $limit
-     * @param int $page
-     */
-    public function setValues(int $total, int $limit, int $page): void {
```

Comment:
Also believe @samerton added this for the infractions module?

Reply:
hm not sure, i did a fast look on infractions module but could not find it, but pretty sure Sam added it for a reason somewhere unless it got removed again

but sam can answer when he see this

Reply:
Not able to find a commit from the same time for Infractions, Resources, Tebex or Voting Plugin, and I can't remember why this was added. I reckon we go with removing it for now and any usages of the Paginator class can amend their implementation